.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -57,23 +60,66 @@ administrator` role:
 
 ## Organization administrator actions
 
-- [ ] Removed the `Azul modify` role (a component of the `operator` role) from
-      the offboarded employee/contractor for all deployments and all zones.
-- [ ] Removed the employee/contractor from the UCSC GitHub organization, or the
-      employee/contractor's employment with the organization has not been
-      terminated.
+- [ ] Removed the offboarded employee/contractor's GitHub account from the
+      `Azul Developers` team in the DataBiosphere organization on GitHub.
+- [ ] Removed the offboarded employee/contractor's Google workspace account from
+      the Google project used for the deployment.
+- [ ] Deleted the Google service account (name derived from the Google workspace
+      account name) in the Google project used for the deployment.
+- [ ] Removed the offboarded employee/contractor's AWS IAM user account in the
+      `gi-gateway` AWS account from the `platform-hca-dev-developer` group (or
+      the `platform-hca-prod-developer` group for a production deployment).
+- [ ] Removed the offboarded employee/contractor's Google workspace account and
+      the Google service account deleted above from the `azul-dev` Terra group.
+
+Additionally, if the employee/contractor is leaving UCSC:
+
+- [ ] Removed the offboarded employee/contractor from the `HumanCellAtlas`
+      GitHub organization.
+- [ ] Removed the offboarded employee/contractor from the `DataBiosphere` GitHub
+      organization.
+- [ ] Removed the offboarded employee/contractor from the `ucsc-cgp` GitHub
+      organization.
+
+Lastly:
+
 - [ ] Assigned ticket to the system administrator.
 
 
 ## System administrator actions
 
-- [ ] Removed the remaining components of the `operator` role from the
-      offboarded employee/contractor.
-- [ ] Removed all remaining roles assigned to the offboarded employee/contractor.
+- [ ] Removed the offboarded employee/contractor's account from the `Azul
+      Operators` team on [GitHub](https://github.com/orgs/DataBiosphere/teams/azul-operators/members).
+- [ ] _____ (TODO: Fill in step(s) to remove `GitLab maintainer` role).
```

Comment:
Is this intended to be left as a TODO?

Reply:
For now, yes. The assignment and removal steps of the `GitLab maintain` role have not yet been documented. (See [source document](https://docs.google.com/document/d/1Kg0dMZmCw6gtkvabD2jYWPZO2Mx_wsC8BJPKdeKTfg0/edit#heading=h.ylrgp0qe635f) heading: `GitLab maintain`)